src/lib/UtapiClient.js

```diff
@@ -207,10 +208,20 @@ export default class UtapiClient {
      * @return {undefined}
      */
     _logMetric(params, method, timestamp, log) {
-        const { bucket, accountId } = params;
-        const logObject = bucket ? { bucket } : { accountId };
-        logObject.method = `UtapiClient.${method}`;
-        logObject.timestamp = timestamp;
+        const { bucket, accountId, user } = params;
+        const logObject = {
+            method: `UtapiClient.${method}`,
+            timestamp,
+        };
+        if (bucket) {
+            logObject.bucket = bucket;
+        }
+        if (accountId) {
+            logObject.accountId = accountId;
+        }
+        if (user) {
+            logObject.user = user;
```

Comment:
We are instead using the userId since the ARN is non-unique (for example, if the user is deleted and recreated, we don't want to get metrics from a previous user).